React/Base/RCTConvert.m

```diff
@@ -1248,9 +1249,9 @@ + (NSArray *)CGColorArray:(id)json
   } else if ([json isKindOfClass:[NSArray class]]) {
     NSMutableArray *mutablePastboardTypes = [NSMutableArray new];
     for (NSString *type in json) {
-      [mutablePastboardTypes addObjectsFromArray:[RCTConvert NSPasteboardType:type]];
-      return mutablePastboardTypes.copy;
+      [mutablePastboardTypes addObjectsFromArray:[RCTConvert NSPasteboardType:type]];      
```

Comment:
nit: We should probably correct this typo while we're here.
`mutablePastboardTypes` -> `mutablePasteboardTypes`

Reply:
Strongly agree on that ;-)